Tasks/AzureRmWebAppDeployment/operations/TaskParameters.ts

```diff
@@ -54,11 +60,36 @@ export class TaskParametersUtility {
 
         return taskParameters;
     }
+
+    private static _initializeDefaultParametersForPublishProfile(taskParameters: TaskParameters): void {
+        taskParameters.PublishProfilePath = tl.getInput('PublishProfilePath', true);
+        taskParameters.PublishProfilePassword = tl.getInput('PublishProfilePassword', true);
+        taskParameters.isLinuxApp = false;
+        taskParameters.isBuiltinLinuxWebApp = false;
+        taskParameters.isContainerWebApp = false;
+        taskParameters.WebAppKind = "webApp";
+        taskParameters.DeployToSlotOrASEFlag = false;
+        taskParameters.ResourceGroupName = null;
+        taskParameters.SlotName = null;
+        taskParameters.VirtualApplication = null;
+        taskParameters.UseWebDeploy = true;
+        taskParameters.RemoveAdditionalFilesFlag = false;
+        taskParameters.SetParametersFile = null;
+        taskParameters.isBuiltinLinuxWebApp = false;
+        taskParameters.ExcludeFilesFromAppDataFlag = false;
+        taskParameters.AdditionalArguments = "";
```

Comment:
Can we add retry flags by default to publish profile too? Since, publish profile is primary used by .NET scenario, it can help in resolving FILE_IN_USE error.

Reply:
done